spec/requests/users_spec.rb

```diff
@@ -1,7 +1,39 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
+RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get user_posts_path(user_id: 1)
+    end
+
+    it 'returns correct status' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the correct template' do
+      expect(response).to render_template('posts/index')
+    end
+
+    it 'returns the correct placeholder text' do
+      expect(response.body).to include('Posts List or here is a list of posts for a given user')
```

Comment:
- [ ] This test also should expect to render the text for the users/index.html.erb file.